src/views/shared/amount-input/amount-input.view.jsx

```diff
@@ -14,15 +14,16 @@ import { getProvider } from "@hermeznetwork/hermezjs/src/providers";
 
 function AmountInput(Component) {
   return function (props) {
-    const { defaultValue, transactionType, account, l2Fee, fiatExchangeRates, preferredCurrency } =
+    const { defaultValue, transactionType, account, fee, fiatExchangeRates, preferredCurrency } =
       props;
     const [gasPrice, setGasPrice] = React.useState(BigNumber.from(0));
     const [value, setValue] = React.useState("");
     const [amount, setAmount] = React.useState({ tokens: BigNumber.from(0), fiat: 0 });
     const [showInFiat, setShowInFiat] = React.useState(false);
-    const [isAmountNegative, setIsAmountNegative] = React.useState(undefined);
-    const [isAmountMoreThanFunds, setIsAmountMoreThanFunds] = React.useState(undefined);
-    const [isAmountCompressedInvalid, setIsAmountCompressedInvalid] = React.useState(undefined);
+    const [isAmountNegative, setIsAmountNegative] = React.useState(false);
+    const [isAmountMoreThanFunds, setIsAmountMoreThanFunds] = React.useState(false);
+    const [isAmountMoreThanWithFees, setIsAmountMoreThanWithFees] = React.useState(false);
```

Comment:
The one that you says is `isAmountMoreThanFunds` this one is more `isAmountLessThanFundsButMoreWithFees`, could we change to that?

Reply:
Right. Here we store if the TX cannot be done because **due to the fee** we are exceeding the funds. What about `areFundsExceededDueToFee`. I think this name clearly explains the point.

I would also consider renaming `isAmountMoreThanFunds` to `isAmountWithFeeMoreThanFunds`. Names are very important and this one is not very precise.